tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/superhero/model/Character.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.graphql.tck.apps.superhero.model;
+
+import org.eclipse.microprofile.graphql.Union;
+
+@Union(name = "Character", description = "Any character known", possibleTypes = {SuperHero.class, Sidekick.class})
```

Comment:
@thejibz I led you astray with my comments in the previous issue/PR.  At our last meeting we discussed that a union should be used for disparate entities, but in this case where we are dealing with `Character`, `SuperHero` and `Sidekick` entities, it would be better to use `Character` as a a GraphQL interface that is implemented by both `SuperHero` and `Sidekick` rather than use unions.

Would you mind changing this PR to implement `@Interface` rather than `@Union`?  The thought is that we might end up doing unions in V2, but the tests that you have should work well for interfaces.

Thanks, Andy

Reply:
Yes I am ok with that change. I agree that it is more of an Interface usecase than Union one. Will do it.

Reply:
Yes I am ok with that change. I agree that it is more of an Interface usecase than Union one. Will do it.